bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -0,0 +1,2078 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="luxtronikheatpump"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="temperatureHeatingCircuitFlow">
+		<item-type>Number:Temperature</item-type>
+		<label>Flow Temp. heating circuit</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperatureHeatingCircuitReturn">
+		<item-type>Number:Temperature</item-type>
+		<label>Return Temp. Heating Circuit</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperatureHeatingCircuitReturnTarget">
+		<item-type>Number:Temperature</item-type>
+		<label>Return Temp. Target Heating Circuit</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperatureBufferTankReturn" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Return Temp. in Buffer Tank</label>
+		<category>Temperature</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="temperatureHotGas" advanced="true">
+		<item-type>Number:Temperature</item-type>
+		<label>Hot Gas Temp.</label>
```

Comment:
I think this looks a bit odd, if you abbreviate, although there is enough space.